helix-core/src/history.rs

```diff
@@ -248,21 +248,27 @@ pub enum UndoKind {
     TimePeriod(std::time::Duration),
 }
 
-const TIME_UNITS: &[&str] = &["seconds", "minutes", "hours", "days"];
+// A subset of sytemd.time time span syntax units.
+const TIME_UNITS: &[(&[&str], &str)] = &[
+    (&["seconds", "second", "sec", "s"], "seconds"),
+    (&["minutes", "minute", "min", "m"], "minutes"),
+    (&["hours", "hour", "hr", "h"], "hours"),
+    (&["days", "day", "d"], "days"),
```

Comment:
Can you put one letter items first?

Reply:
I think it's fine either way, I just don't feel it matters much so I'm not inclined to change it. I copied it in the order that it was listed in the spec.